common/src/main/java/com/datastrato/gravitino/json/JsonUtils.java

```diff
@@ -237,8 +238,13 @@ private static class ObjectMapperHolder {
   /**
    * Get the shared ObjectMapper instance for JSON serialization/deserialization.
    *
-   * @return The ObjectMapper instance.
+   * <p>This instance should only be used for testing. For production or other environments, obtain
+   * an {@link ObjectMapper} from the appropriate {@code ObjectMapperProvider} on the server or
+   * client side.
+   *
+   * @return the shared {@link ObjectMapper} instance for tests.
```

Comment:
I think your comments should be differentiated between server and client. Currently, it is the same.

Reply:
fixed